test/integration/short/pool-simulator-tests.js

```diff
@@ -8,7 +8,7 @@ const utils = require("../../../lib/utils");
 const types = require("../../../lib/types");
 const policies = require("../../../lib/policies");
 const promiseUtils = require("../../../lib/promise-utils");
-const { version } = require("../../../index");
+const { version } = require("../../../main");
```

Comment:
Hmm? Why is this change needed?

Reply:
We renamed index from datastax to main, as index is reserved by napi

Reply:
Just to be sure: won't this break drop-in compatibility?

Reply:
That's fully correct